lighthouse-cli/test/smokehouse/byte-efficiency/expectations.js

```diff
@@ -28,15 +30,14 @@ module.exports = [
         score: '<100',
         extendedInfo: {
           value: {
-            wastedKb: 14,
+            wastedKb: 45,
```

Comment:
as discussed, can we add more `wastedMs` values to attempt to catch regressions on our side or Chrome's?

Reply:
done